onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java

```diff
@@ -259,44 +266,62 @@ private void changePreferenceSummary(String preferenceKey) {
         if (preferenceKey.equalsIgnoreCase(getString(R.string.preference_key_region))
                 || preferenceKey.equalsIgnoreCase(getString(R.string.preference_key_oba_api_url))) {
             if (Application.get().getCurrentRegion() != null) {
-                regionPref.setSummary(Application.get().getCurrentRegion().getName());
-                customApiUrlPref
+                mPreference.setSummary(Application.get().getCurrentRegion().getName());
+                mCustomApiUrlPref
                         .setSummary(getString(R.string.preferences_oba_api_servername_summary));
+                String customOtpApiUrl = Application.get().getCustomOtpApiUrl();
+                if (customOtpApiUrl == null) {
```

Comment:
Yes, I'm asking about line 273, based on how line 289 looks:

Line 289:
~~~
 +            if (!TextUtils.isEmpty(otpApiUrl)) {
 +                mCustomOtpApiUrlPref.setSummary(otpApiUrl);
 +            } else {
 +                mCustomOtpApiUrlPref.setSummary(
 +                        getString(R.string.preferences_otp_api_servername_summary));
 +            }
~~~

Line 273:
~~~
+                if (customOtpApiUrl == null) {
+                    mCustomOtpApiUrlPref
 +                            .setSummary(getString(R.string.preferences_otp_api_servername_summary));
 +                } else {
 +                    mCustomOtpApiUrlPref.setSummary(customOtpApiUrl);
 +                }
~~~

Seems like these are attempting doing the same thing, but one is using a null check, and the other is using `TextUtils.isEmpty()`.  So I believe these should both be the same, unless I'm missing something.